staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/customresource_handler.go

```diff
@@ -571,6 +664,35 @@ func (r *crdHandler) tearDown(oldInfo *crdInfo) {
 		// destroy only the main storage. Those for the subresources share cacher and etcd clients.
 		storage.CustomResource.DestroyFunc()
 	}
+
+	r.updateActiveTeardownsCount(oldInfo.name, -1)
```

Comment:
Consider moving this to the top of the function and using a defer method

Reply:
Done